app/javascript/mastodon/locales/fr.json

```diff
@@ -145,7 +145,7 @@
   "onboarding.page_six.apps_available": "De nombreuses {apps} sont disponibles pour iOS, Android et autres. Et maintenant… Bon appouétit !",
   "onboarding.page_six.github": "Mastodon est un logiciel libre, gratuit et open-source. Vous pouvez rapporter des bogues, suggérer des fonctionnalités, ou contribuer à son développement sur {github}.",
   "onboarding.page_six.guidelines": "règles de la communauté",
-  "onboarding.page_six.read_guidelines": "S’il vous plaît, n’oubliez pas de lire les {guidelines} !",
+  "onboarding.page_six.read_guidelines": "S’il vous plait, n’oubliez pas de lire les {guidelines} !",
```

Comment:
Both are fine, "s'il vous plait" or "s'il vous plaît". No needs to change here, unless you're looking towards consistency across the whole translations. If it would be the case, please let us know.

Reply:
It's just to follow the *[Rectifications orthographiques du français en 1990](https://fr.wikipedia.org/wiki/Rectifications_orthographiques_du_fran%C3%A7ais_en_1990)* where the circumflex accent is removed on a lot of words (even if both forms are accepted).

Reply:
The 1990 dictionary is "proposing" an new form for this (and other) word, not "imposing". Looking quickly at the FR translation here, there are other "plaît" but no "plait".

Hence, I'd be inclined to leave "plaît" as it is, with the circumflex accent.